app/models.py

```diff
@@ -1589,6 +1589,44 @@ def serialize_for_csv(self):
 
         return serialized
 
+    def serialize_permanent_failure(self):
+        return {
+            "billing_code": self.billing_code,
+            "body": None,
```

Comment:
we have the body (the content of the sms), is there a reason we wouldn't return it?

Reply:
Per our conversation with you, Dave and Kyle. Storing only incoming data. Body will be added by GET